docker/api/volume.py

```diff
@@ -45,6 +45,9 @@ def create_volume(self, name=None, driver=None, driver_opts=None,
             driver (str): Name of the driver used to create the volume
             driver_opts (dict): Driver options as a key-value dictionary
             labels (dict): Labels to set on the volume
+            subpath (str): Subpath within the volume to use as the volume's
```

Comment:
I'd change this to 
```
subpath (str): Path inside a volume to mount instead of the volume root.
```
To better match documentation we have elsewhere (eg https://github.com/docker/docs/blob/23f3fbd0639a24d0e2280fd0edb7f72b3f0cb1ca/content/reference/compose-file/services.md?plain=1#L1865)

@dvdksn WDYT?

Reply:
SGTM